addons/l10n_no_edi/data/ehf_3_template.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <template id="export_ehf_3_invoice_partner"
+                  inherit_id="account_edi_ubl_bis3.export_bis3_invoice_partner"
+                  primary="True">
+            <xpath expr="//*[local-name()='PartyTaxScheme']" position="after">
+                <cac:PartyTaxScheme xmlns="urn:oasis:names:specification:ubl:schema:xsd:Invoice-2"
+                xmlns:cac="urn:oasis:names:specification:ubl:schema:xsd:CommonAggregateComponents-2"
+                xmlns:cbc="urn:oasis:names:specification:ubl:schema:xsd:CommonBasicComponents-2">
+                    <cbc:CompanyID>Foretaksregisteret</cbc:CompanyID>
+                    <cac:TaxScheme>
+                        <cbc:ID>TAX</cbc:ID>
+                    </cac:TaxScheme>
+                </cac:PartyTaxScheme>
```

Comment:
This could be added in generic when the withholding tax task will be merged since there will be a `family='vat'` field on taxes I think

Reply:
Yep !